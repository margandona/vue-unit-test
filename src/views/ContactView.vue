<template>
    <div class="contact">
      <h1>This is the Contact page</h1>
    </div>
  </template>
  
  <script>
  export default {
    name: 'ContactView'
  }
  </script>
  
  <style scoped>
  .contact {
    text-align: center;
  }
  </style>
  