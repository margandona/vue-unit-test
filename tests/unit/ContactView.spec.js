import { shallowMount } from '@vue/test-utils'
import ContactView from '@/views/ContactView.vue'

describe('ContactView.vue', () => {
  it('renders the Contact page', () => {
    const wrapper = shallowMount(ContactView)
    expect(wrapper.text()).toMatch('This is the Contact page')
  })

  it('matches snapshot', () => {
    const wrapper = shallowMount(ContactView)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
