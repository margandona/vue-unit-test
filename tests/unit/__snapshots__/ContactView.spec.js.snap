// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView.vue matches snapshot 1`] = `
<div class="contact">
  <h1>This is the Contact page</h1>
</div>
`;
